test/modules/linear_boltzmann_solvers/transport_steady_cyl/transport_2d_cyl_2_multigroup.lua

```diff
@@ -60,7 +60,7 @@ mg_src = xs.IsotropicMultiGroupSource.FromArray(source)
 material0:SetIsotropicMGSource(mg_src)
 
 -- Setup Physics
-pquad0 = aquad.CreateCylindricalProductQuadrature(GAUSS_LEGENDRE_CHEBYSHEV, 4, 8)
```

Comment:
in RZ, why are the integers passed in arguments the same before and after your changes? I thought that the number of polar angles was, previously, the number of positive angles and now it is the total number of levels. I was expecting a factor 2. Maybe a similar question for the second entry as well.

Reply:
They shouldn't be. I just accidentally missed them in my mass editing of the inputs. I've fixed those inputs. Nice catch!

Reply:
i didn't scrutinize every input file. should i do that or will the regression test catch the issues? it's a pain to have to go through all those input files, for you as author of the PR, for me as reviewer. if only the original developer talked to experts to learn what the convention in the field was ...

Reply:
I have a problem with this. none of the 3 tests that failed is this input file. so the regression suite didn't not catch that input change because the results was unaffected. that's a bad test!

Reply:
It is a pain. And I went through them twice and still missed some. I modified them a test directory at a time, so that's likely why I missed these. I'll go back through all of them a third time just to make sure I got them all... and make the correct changes.

Reply:
> I have a problem with this. none of the 3 tests that failed is this input file. so the regression suite didn't not catch that input change because the results was unaffected. that's a bad test!

I agree. We have #124 open to address this.